### ISSUE TITLE
Product Branding

### ISSUE THREAD

**Issue by mwtsn** [mkdo/cpd#2]
Work on product branding to be completed by the team and @mkjonesuk. We have suggested Butterfly CPD as a product brand name, conversation details here:

"At the moment 'Butterfly CPD' is the frontrunner as a name with either of these as a tagline:
Take the pain out of CPD

A free, open & easy to use CPD system build for WordPress

This is very early work of course but we feel this is on the right track. The use of a butterfly logo has also been indicated which we can merge into a 'book' or 'journal' type icon to sum up the systems purpose as a product."

"Our recommendaion is 'Butterfly' which I know sounds a little 'soft' but it's also friendly and has obvious links to the metamorphosis process that CDP is designed to capture."

Currently @chri55mith is tabling the Branding options with the team and will get back to us.

**Comment by chri55mith** [comment]
Prefer "A free, open & easy to use CPD system build for WordPress" as the strap line.

**Comment by chri55mith** [comment]
Not sure about Butterfly CPD, Geoff, Ian, Joan what do you think? Matt can you assign all members to this discussion get their thoughts on this?

**Comment by mwtsn** [comment]
@chri55mith Geoff has an invite already, just waiting on @mkjonesuk to invite Joan.

**Comment by IanWraith** [comment]
Could someone send Geoff another invite ? I spoke to him briefly today and
he isn't getting any emails from Github about these discussions. If he is
already on and there is a problem let me know and I will look into his
Github setup.

Ian

**Comment by mkjonesuk** [comment]
Hey @mkdo/cpd-partners I've checked the invites and Geoff is indeed still pending - Joan will need to sign up for a username first and then we can add her.

Hope that's all clear :)

**Comment by joanward** [comment]
Hi All
My username is joanward (signed in via joanward@istonline.org.uk)
I'm not sure about the 'butterfly' brand - that doesn't say focus or commitment to me, sorry. I get the suggested analogy in terms of metamorphosis but when I think of a butterfly I think of something that flits from one thing to another haphazardly.

**Comment by mwtsn** [comment]
Hi @joanward welcome aboard, good to have you here. 

Point taken about Butterfly, looks like it isn't that popular. Another one of my favourites was "Monarch CPD", I don't know if you think the same way about that or not.

If not @mkjonesuk and I will get back to the drawing board.

Also while you are here @joanward, issue #14 specifically could do with some of your knowledge.

Thanks for your input.

**Comment by mkjonesuk** [comment]
Hey @joanward probably best that we go back to the workshop report to tie down a better branding solution.

If you check page 2 of the 'overview' we have the outcomes, here's a selection:

**Keywords & phrases:**
Easy to use
Logging information
Personal Development
Training and Development
Helping users
Capture data
Professional 
Easy to find information
Webfolio
Free

**Branding guide:**
Develop
Mature
Evolve
Change
Log data
Transform
Update

The last page of the report has other suggested names and taglines, as we have a consensus on the tagline we just need to go back to the following for inspiration:

**System name:**
SentinelCPD
Butterfly*
Butterfly CPD*
CPDPress 
Monarch
Development Dashboard
EasyCPD
CPDFolio
SmartCPD

The **CPDPress** item was discounted as it lay too heavy on the WordPress branding. But the rest are still good to focus on and improve. 

One other factor is domain names, **butterflycpd.com** is currently available which was another reason this was highlighted as a good option.

Moving forward we should also check trademarks and business names.

With this in mind let's open up a new debate around the brand name and see what we can get from the information in the report.

**Comment by chri55mith** [comment]
It's a bit sterile but quite like development dashboard.

**Comment by joanward** [comment]
I also quite like Development Dashboard, along with SmartCPD.

**Comment by mkjonesuk** [comment]
@chri55mith agree, it's a little bit to the point. A brand name for a product like this should be more snappy. Take some other system names like SharePoint, Magnolia, Joomla, Moodle, Plone etc. Easy to focus in and remember. 

The idea of the workshop was to take the outcome of the feedback and discussions and use the keywords and phrases to come up with suitable options. 

If none of the options in the report are suitable then we'll have to do another round of creative planning. This time however we can use the comments here to fill in the blanks.

Keep throwing around ideas and we'll start a new list.

**Comment by mkjonesuk** [comment]
@mkdo/cpd-partners - I'm just following up with some extra notes around this. As we have no further comments I've taken some of @joanward's feedback and put together a new list of candidates:

Balance CPD
Smarter CPD
SkyRocketCPD (or just SkyRocket)
Aspire CPD (I really like this one)

All of the above have *decent* domain names available. So let's narrow this down and we can start to pull together a branded information site to help grow participants.

**Comment by mwtsn** [comment]
@IanWraith @chri55mith @el1gxh do you have any input on the above comment by @mkjonesuk? Aspire CPD could be a winner.

**Comment by joanward** [comment]
Hi
I quite like AspireCPD also - but to throw another couple of suggestions into the pot what about CatalystCPD or ProCPD

**Comment by el1gxh** [comment]
I like the Aspire CPD. It's catchy but I'm not sure if we should have something about the catalyst in there somewhere.

**Comment by mwtsn** [comment]
It sounds like Aspire CPD could be a winner @chri55mith do you have any further thoughts on this? 

@mkjonesuk I think I have a brand in mind for this. Hang on, I'll mock something up quickly (bare in mind I'm no graphic designer, but its just a suggestion):

![fullsizerender](https://cloud.githubusercontent.com/assets/2066205/7294254/397cb15c-e9a3-11e4-9078-290fb1c34f98.jpg)

The A of Aspire is represented by the large mountain in the mountain range, and the mountains represent the Aspiration of the participant.

Cool blues, greys and whites would serve this brand well.

**Comment by joanward** [comment]
I quite like the concept but wonder whether it would look better with 3 decreasing height peaks, rather than 2?

**Comment by mwtsn** [comment]
@joanward my drawing is a little crude I'm afraid, but suggestion taken on board. I'll leave it to @mkjonesuk to come up with a better option. He's the expert at these things.

**Comment by mwtsn** [comment]
Here is another attempt, try not to judge me too much:

![fullsizerender](https://cloud.githubusercontent.com/assets/2066205/7296481/edaa0288-e9b7-11e4-8b82-0cbefd836bc1.jpg)

**Comment by mkjonesuk** [comment]
Hi all,

I'm on with other projects today so will catch up with this tomorrow - glad
to see Aspire being considered, the logo ideas line up with my own so if we
can agree this is indeed the one we all like I can start basic branding
next week :)

**Comment by joanward** [comment]
I prefer the mockup using 3 peaks - but just to throw a spanner in the works, the logo mockup suggests SummitCPD to me rather than AspireCPD....?

**Comment by mkjonesuk** [comment]
Hi @joanward - we feel that 'Summit' sounds a little too corporate, like a Microsoft or IMB product (think servers, performance, etc). 

On the 'Aspire CPD' front - I put together a more mature mock up of the logo:

![aspirecpd](https://cloud.githubusercontent.com/assets/1765467/7373251/34da88bc-edc3-11e4-8f6d-24e06826df66.png)

Any feedback welcome, this is just a first draft.

**Comment by el1gxh** [comment]
I do like Aspire but agree with @joanward, the 3 peaks represent a summit  We used 3 ladders with people climbing the ladders can this be modified? @chri55mith

**Comment by mkjonesuk** [comment]
Hey Geoff,

Not keen on the ladders. Feels a bit 80s DIY advert.

Agree to some extent that the 'mountains' are confusing. We'll work on
something else around the Aspire theme.

**Comment by el1gxh** [comment]
@mkjonesuk your right, it's a difficult one, it might be better with some curve lines and a few stars placed within. I sure you'll come up with something that fits.

**Comment by mwtsn** [comment]
Hi @el1gxh @joanward @mkjonesuk,

A logo should be very easy to recognise, that can work with many colour schemes (eg work as a two tone pallet) and resolutions, so it needs to be something very simple graphically. The best examples of logos like this that I can give are the Apple logo the MacDonalds 'M', or the new Microsoft Logo.

To Aspire is an abstract concept, mountain peaks may make some people think of a summit, but ladders would equally make some think of DIY. So finding something to fit would be tricky.

My suggestion would be to stick with the current concept, but perhaps make it more abstract?

I have an idea for an abstract mountain concept. I'll share later tonight when I get chance.

**Comment by el1gxh** [comment]
@mkjonesuk you guys do this for a living so I know what ever you put together will be great. I totally agree about the ladders it's a bit DIY. It was just a thought.

**Comment by joanward** [comment]
PeakCPD would fit with the existing logo mockup - tell me to shut up when you're tired of me throwing suggestions into the pot!!

**Comment by mwtsn** [comment]
Hi @joanward, I think name 'Aspire' gives a clue about the system's purpose, where as peak is a little too far removed from what we were trying to achieve. 

I'm sure with a new logo concept things will fall into place.

**Comment by mwtsn** [comment]
OK, as promised, here is my 'abstract' mountain idea for the 'Aspire' concept.

The idea is sort of like an upside down diamond shape, which is an abstract of three peaks. The thinking behind the logo, is the diamond shape is a 'cut out' meaning that it can be filled with any abstract colours or designs, but the shape itself is the key concept. 

Very rough, but here it is:

![fullsizerender](https://cloud.githubusercontent.com/assets/2066205/7388445/0c11b654-ee5d-11e4-84b4-f8ef1b2d7988.jpg)

**Comment by mkjonesuk** [comment]
I like this new concept and agree - the tail should not wag the dog, I think there is a general agreement that 'Aspire CPD' is the strongest brand name we have so we need to apply a branding solution to this.

I will work on maturing these ideas with images next chance I get.
